private/bufpkg/bufimage/bufimagemodify/internal/internal.go

```diff
@@ -59,19 +59,28 @@ var (
 	// RubyPackagePath is the SourceCodeInfo path for the ruby_package option.
 	// https://github.com/protocolbuffers/protobuf/blob/61689226c0e3ec88287eaed66164614d9c4f2bf7/src/google/protobuf/descriptor.proto#L453
 	RubyPackagePath = []int32{8, 45}
+	// JSTypePackageSuffix is the SourceCodeInfo sub path for the jstype field option.
+	// https://github.com/protocolbuffers/protobuf/blob/61689226c0e3ec88287eaed66164614d9c4f2bf7/src/google/protobuf/descriptor.proto#L567
+	JSTypePackageSuffix = []int32{8, 6}
```

Comment:
I was mostly commenting because of `Package`. I agree that it would be nice to make it clear from the name that it is not a full path. `JSTypeSubPath` would do that for me.

Reply:
Oh yes, `Package` was definitely a mistake. Somehow I missed it twice...
> JSTypeSubPath would do that for me.

Sounds good.